lms/static/sass/base/_variables.scss

```diff
@@ -82,6 +82,27 @@ $gray-d2: shade($gray,40%); // #4c4c4c
 $gray-d3: shade($gray,60%); // #323232
 $gray-d4: shade($gray,80%); // #191919
 
+$cms-blue: rgb(0, 159, 230); // NOTE - eventually, this set of blues should be synced with CMS.  
```

Comment:
@frrrances That make sense.  I'll go through and update the colors here to go back to using m-blue-*. We can treat this as a separate cleanup task.